From 57547e634be6a12ce64f1a1ce5c3c72ddbc33e80 Mon Sep 17 00:00:00 2001
From: Kei Okada <k-okada@jsk.t.u-tokyo.ac.jp>
Date: Wed, 26 Aug 2020 14:05:55 +0900
Subject: [PATCH 2/2] dh_strip: exclude flac for non x86 environment

---
 debian/rules.em | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/debian/rules.em b/debian/rules.em
index a23ba41792..0bd4374d3e 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -69,3 +69,6 @@ override_dh_auto_install:
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
 	dh_auto_install
+
+override_dh_strip:
+	dh_strip $@ -Xflac
-- 
2.17.1

