From 3e2bd5e8351621a780d9e6f4009c3a07537ea596 Mon Sep 17 00:00:00 2001
From: Kei Okada <k-okada@jsk.t.u-tokyo.ac.jp>
Date: Wed, 9 Jun 2021 18:10:19 +0900
Subject: [PATCH 3/4] exclude flac for non x86 evironment

---
 debian/rules.em | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 0bd4374d3..8d5f4d209 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -71,4 +71,5 @@ override_dh_auto_install:
 	dh_auto_install
 
 override_dh_strip:
-	dh_strip $@ -Xflac
+	# WORKAROUND FOR FIX https://github.com/tork-a/jsk_3rdparty-release/commit/302452be190f0423c4e65be401513f27cbca95ad#commitcomment-51919365
+	dh_strip $@ -Xflac-x86_64
-- 
2.17.1

